src/transformers/training_args.py

```diff
@@ -1721,8 +1722,10 @@ def __post_init__(self):
             for fsdp_option in self.fsdp:
                 if fsdp_option.upper() in FSDP_SHARDING_STRATEGY:
                     # set environment variable for FSDP sharding strategy
-                    os.environ[f"{prefix}SHARDING_STRATEGY"] = str(
-                        FSDP_SHARDING_STRATEGY.index(fsdp_option.upper()) + 1
+                    os.environ[f"{prefix}SHARDING_STRATEGY"] = (
+                        str(FSDP_SHARDING_STRATEGY.index(fsdp_option.upper()) + 1)
+                        if version.parse(accelerate_version) >= version.parse("0.26.0")
```

Comment:
`is_accelerate_available` already takes in a `min_version` so we can simplify this
```suggestion
                        if is_accelerate_available("0.26.0")
```

Reply:
Done